docker/ci/builder/Dockerfile

```diff
@@ -1,26 +1,28 @@
-FROM java:openjdk-8-alpine
+FROM openjdk:11-jdk-slim
 
 ARG USER_NAME
 ARG USER_ID
 ARG GROUP_NAME
 ARG GROUP_ID
 
-RUN apk update \
-  && apk add bash ca-certificates wget git openssh \
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends bash ca-certificates wget git openssh-client \
```

Comment:
`apt-get`ien perään on hyvä lisätä `rm -rf /var/lib/apt/lists/*` mikä tyhjää pakettiasennusten kakun. (Ks esim. https://hackernoon.com/tips-to-reduce-docker-image-sizes-876095da3b34)